relay-server/src/actors/outcome.rs

```diff
@@ -599,10 +594,10 @@ impl Handler<TrackRawOutcome> for HttpOutcomeProducer {
     type Result = Result<(), OutcomeError>;
     fn handle(&mut self, message: TrackRawOutcome, ctx: &mut Self::Context) -> Self::Result {
         if self.config.emit_outcomes() {
-            self.send_http_message(message, ctx)
-        } else {
-            Ok(()) // processing not enabled and emit_outcomes disabled
```

Comment:
just wondering how this comment changes

Reply:
The else block is gone essentially, but the same applies implicitly.